content/patterns/checkbox/examples/checkbox-mixed.html

```diff
@@ -18,103 +18,104 @@
         </ul>
       </section>
 
       <section>
         <div class="example-header">
-          <h2 id="ex_label">Example</h2>
+          <h2 id="ex_label">例</h2>
         </div>
-        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_label ex_start_sep" aria-label="の開始"></div>
         <div id="ex1">
           <fieldset class="checkbox-mixed">
-            <legend>Sandwich Condiments</legend>
-            <div role="checkbox" class="group_checkbox" aria-checked="mixed" aria-controls="cond1 cond2 cond3 cond4" tabindex="0">All condiments</div>
+            <legend>サンドイッチの調味料</legend>
+            <div role="checkbox" class="group_checkbox" aria-checked="mixed" aria-controls="cond1 cond2 cond3 cond4" tabindex="0">すべての調味料</div>
             <ul class="checkboxes">
               <li>
-                <label><input type="checkbox" id="cond1">Lettuce</label>
+                <label><input type="checkbox" id="cond1">レタス</label>
               </li>
               <li>
-                <label><input type="checkbox" id="cond2" checked>Tomato</label>
+                <label><input type="checkbox" id="cond2" checked>トマト</label>
               </li>
               <li>
-                <label><input type="checkbox" id="cond3">Mustard</label>
+                <label><input type="checkbox" id="cond3">マスタード</label>
               </li>
               <li>
-                <label><input type="checkbox" id="cond4">Sprouts</label>
+                <label><input type="checkbox" id="cond4">スプラウト</label>
               </li>
             </ul>
           </fieldset>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
-          <li>To help assistive technology users understand that the checkboxes are all part of a group of related checkboxes named <q>Sandwich Condiments</q>, the checkboxes are wrapped in a <code>fieldset</code> element which is labeled by a <code>legend</code> element.</li>
-          <li>To make it easier to perceive that clicking either the label or checkbox will activate the checkbox, when a pointer hovers over either the checkbox or label, the background color changes, a border appears, and the cursor changes to a pointer.</li>
+          <li>支援技術の利用者が、チェックボックスがすべて「サンドイッチの調味料」という名前の関連チェックボックスのグループの一部であることを理解できるように、チェックボックスは <code>legend</code> 要素でラベル付けされた <code>fieldset</code> 要素でラップされています。</li>
+          <li>ラベル又はチェックボックスのいずれかをクリックするとチェックボックスがアクティブになることを知覚しやすくするために、ポインターがチェックボックス又はラベルのいずれかにホバーすると、背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
```

Comment:
activateは「動作(する|させる)」
```suggestion
          <li>ラベル又はチェックボックスのいずれかをクリックするとチェックボックスが動作することを知覚しやすくするために、ポインターがチェックボックス又はラベルのいずれかにホバーすると、背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
```